### PR TITLE
[injected-wallets] : feat - add Exodus wallet

### PR DESCRIPTION
**Pull request by bulgakovk**
### Description

Adds support for Exodus Browser Extension

### Checklist
- [x] The version field in `package.json` is incremented following [semantic versioning](https://semver.org/)
- [x] The box that allows repo maintainers to update this PR is checked
- [x] I tested locally to make sure this feature/fix works
- [ ] This PR passes the Circle CI checks

### CODE REVIEW

File: packages/injected/package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@web3-onboard/injected-wallets",
-  "version": "2.0.7",
+  "version": "2.1.0",
```
Comment (Adamj1232) [inline packages/injected/package.json:3]:
```suggestion
  "version": "2.0.8-alpha.1",
```

**Comment by bulgakovk** [comment]
> @bulgakovk Hey! Could you update this PR with latest develop? Then we can get it merged in 👍🏾

Hey sure thing! Just rebased on the latest v2-develop branch

File: packages/injected/package.json [hunk 2]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@web3-onboard/injected-wallets",
-  "version": "2.0.8",
+  "version": "2.0.8-alpha.1",
```
Comment (taylorjdawson) [inline packages/injected/package.json:3]:
```suggestion
  "version": "2.0.9-alpha.1",
```

**Review by taylorjdawson — CHANGES_REQUESTED** [review]
Just version bump this and we're good!

**Review by taylorjdawson — APPROVED** [review]

**Pull request merged**